pypesto/ensemble/ensemble.py

```diff
@@ -599,9 +604,21 @@ def from_sample(
                 geweke_test(result)
             burn_in = result.sample_result.burn_in
             x_vectors = x_vectors[burn_in:]
+        else:
+            burn_in = 0
+
+        # added cutoff
+        if f_quantile is None:
+            pass
+        else:
+            x_vectors = calculate_hpd(
+                result=result, burn_in=burn_in, ci_level=f_quantile
```

Comment:
Why `f_quantile` here but `ci_level` in `calculate_hpd`? Same terms everywhere would be preferable